app/components/file_attachment_list/file_attachment_document.js

```diff
@@ -179,7 +179,8 @@ export default class FileAttachmentDocument extends PureComponent {
                 OpenFile.openDoc([{
                     url: `${prefix}${path}`,
                     fileName: file.name,
-                    fileType: file.extension
+                    fileType: file.extension,
+                    cache: false
```

Comment:
https://github.com/mattermost/mattermost-mobile/pull/1176

Reply:
we don't need it as we are saving the file already with RNFetchBlob

Reply:
Cool